packages/client-sdk-nodejs/src/internal/cache-data-client.ts

```diff
@@ -242,10 +243,12 @@ export class CacheDataClient implements IDataClient {
           } else if (newState === ConnectivityState.CONNECTING) {
             // The connection goes through the CONNECTING state before becoming READY,
             // so we must watch it twice.
+            this.logger.debug(`Connecting! Current state: ${newState}`);
             this.connectWithinDeadline(deadline).then(resolve).catch(reject);
           } else {
             this.logger.error(
-              `Unable to connect to Momento: Unexpected connection state: ${newState}. Please contact Momento if this persists.`
+              `Unable to connect to Momento: Unexpected connection state: ${newState}., oldState: ${currentState}
+              Please contact Momento if this persists.`
             );
             resolve();
```

Comment:
these look good; any reason not to add a log in the clientfactoryfn line 168? or in the connect fn line 193?

Reply:
the one at 168 gets logged as a gRPC trace saying `Channel constructed` etc. but either way added at both places.

Reply:
thanks! i don't necessarily want to always have to rely on grpc trace logs so it's good to have as much as possible covered in our own.